code/modules/client/client_procs.dm

```diff
@@ -401,11 +338,9 @@
 
 
 	if(prefs.toggles & PREFTOGGLE_DISABLE_KARMA) // activates if karma is disabled
-		if(establish_db_connection())
```

Comment:
Why do you not need this anymore?

Reply:
We already have a DB connection if we managed to load the preference toggles, no point in trying to establish something that already exists .